infrastructure/lib/kit-infrastructure.ts

```diff
@@ -46,6 +47,8 @@ export class KITInfrastructure extends Stack {
       clusterName: id,
       vpc: vpc,
       role: clusterRole,
+      mastersRole: clusterRole,
```

Comment:
I have this change locally. If you are going with this change, you might also need to add `sts assume role` permissions for this role for the account where it's created using `composite IAM principal ` stuff.

Reply:
I can drop this part.